app/helpers/checkout_helper.rb

```diff
@@ -74,8 +74,9 @@ def display_checkout_taxes_hash(order)
       {
         amount: Spree::Money.new(tax_amount, currency: order.currency),
         percentage: number_to_percentage(tax_rate.amount * 100, precision: 1),
+        rate_amount: tax_rate.amount,
       }
-    end
+    end.sort_by { |tax| tax[:rate_amount] }
```

Comment:
Great! Thanks.

I was expecting this change to [break this assertion](https://github.com/openfoodfoundation/openfoodnetwork/blob/8270a3f3868d031a0ed1eceef05659803c9d0db5/spec/system/admin/invoice_print_spec.rb#L259) - but it does not, which means this system spec can be improved to account for the sorting of tax rates. I've addressed it in a separate PR, in this commit 080b2ce9816471c8e2828b58117368a4c7731e01.